Dispatcher/Repositories/ArticlesRepository.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ArticlesRepository.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 30/04/2023.
+//
+
+import Foundation
+
+class ArticlesRepository {
+
+    let alamofireManager = AlamofireManager()
+    var articles: [Article] = []
```

Comment:
No need to save the articles in the repository as we pass them back to the viewModel